package.json

```diff
@@ -26,7 +26,8 @@
     "chromatic": "chromatic",
     "commit": "cz",
     "plop": "plop",
-    "pkg:aio": "yarn workspace @kaizen/components"
+    "pkg:aio": "yarn workspace @kaizen/components",
+    "build:deps": "yarn workspace @kaizen/design-tokens build && yarn workspace @kaizen/tailwind build"
```

Comment:
Make sure to fix the storybook/release scripts :)

Actually, since building the deps is only required for releasing the AIO, could you move this into the AIO's `prepublish`?

Reply:
Ah good idea, can do! I only put this here to help me develop lol